app/build.gradle

```diff
@@ -73,8 +73,12 @@ android {
 
 dependencies {
     // Mapbox Navigation SDK
-    implementation "com.mapbox.navigation:android:2.10.1"
-    implementation "com.mapbox.navigation:ui-dropin:2.10.1"
+    implementation "com.mapbox.navigationcore:navigation:3.0.0"
+    implementation "com.mapbox.navigationcore:voice:3.0.0"
+    implementation "com.mapbox.navigationcore:copilot:3.0.0"
+    implementation "com.mapbox.navigationcore:ui-maps:3.0.0"
+    implementation "com.mapbox.navigationcore:ui-components:3.0.0"
```

Comment:
```suggestion
    implementation "com.mapbox.navigationcore:navigation:3.0.2"
    implementation "com.mapbox.navigationcore:voice:3.0.2"
    implementation "com.mapbox.navigationcore:copilot:3.0.2"
    implementation "com.mapbox.navigationcore:ui-maps:3.0.2"
    implementation "com.mapbox.navigationcore:ui-components:3.0.2"
```

Reply:
Do we need all 5 deps? Why not `com.mapbox.navigationcore:navigation:3.0.2` only?

Reply:
Hmm, I see that we'd also need to add `implementation "com.mapbox.navigationcore:ui-components:3.0.2"` which is not part of the primary artifact. This means that we're not including the UI components in the main artifact anymore, was that intentional? We're not mentioning this in any of the migration docs pages or documentation for UI components in https://docs.mapbox.com/android/navigation/guides/ui-components/.

Reply:
Discussed internally, that we're good to keep ui-components in separate from the main artefact. I'll also update docs!